frontend/projects/ui/src/app/components/toast-container/refresh-alert/refresh-alert.service.ts

```diff
@@ -9,9 +9,10 @@ import { ConfigService } from '../../../services/config.service'
 // Watch for connection status
 @Injectable({ providedIn: 'root' })
 export class RefreshAlertService extends Observable<boolean> {
-  private readonly stream$ = this.patch
-    .watch$('server-info', 'version')
-    .pipe(map(version => !!this.emver.compare(this.config.version, version)))
+  private readonly stream$ = this.patch.watch$('server-info', 'version').pipe(
+    map(version => !!this.emver.compare(this.config.version, version)),
+    endWith(false),
```

Comment:
Upon logout subscriptions to `patch.watch$` are terminated and with `endWith(false)` we can close toasts.